boefjes/boefjes/sql/config_storage.py

```diff
@@ -173,6 +173,31 @@ def get_enabled_normalizers(self, organisation_id: str) -> list[str]:
 
         return [plugin[0] for plugin in enabled_normalizers.all()]
 
+    def get_disabled_boefjes(self, organisation_id: str) -> list[str]:
+        enabled_boefjes = (
+            self.session.query(BoefjeInDB.plugin_id)
+            .join(BoefjeConfigInDB)
+            .filter(BoefjeConfigInDB.boefje_id == BoefjeInDB.id)
+            .join(OrganisationInDB)
+            .filter(BoefjeConfigInDB.organisation_pk == OrganisationInDB.pk)
+            .filter(OrganisationInDB.id == organisation_id)
+            .filter(BoefjeConfigInDB.enabled == False)
+        )
+        return [x[0] for x in enabled_boefjes.all()]
```

Comment:
Since you're not doing anything specifically with the `Row` objects, you can also use `scalars()` to get the elements you need. I think this is also cleaner because it gives you a list of values without tuple unpacking.

```suggestion
        return enabled_boefjes.scalars().all()
```

I've seen this in other places, but it just got my attention now. We can choose to optimize or clean it up in a future PR. I'll leave this up to you.